### PR TITLE
Payments

### PR DESCRIPTION
**Pull request by axshani**
Implemented Payments resource

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@unit-finance/unit-node-sdk",
-  "version": "0.2.16",
+  "version": "0.4.0",
```
Comment (Elroie) [inline package.json:3]:
```suggestion
  "version": "0.2.16",
```

**Review by Elroie — CHANGES_REQUESTED** [review]

**Review by ilyamerman — APPROVED** [review]

**Review by Elroie — APPROVED** [review]

File: types/payments.ts [hunk 2]
```diff
@@ -0,0 +1,331 @@
+import { Relationship, Counterparty } from "./common"
+
+type PaymentStatus = "Pending" | "Rejected" | "Clearing" | "Sent" | "Canceled" | "Returned"
+
+export type Payment = ACHPayment | BookPayment
```
Comment (ilyamerman) [inline types/payments.ts:5]:
```suggestion
export type Payment = AchPayment | BookPayment
```

File: types/payments.ts [hunk 3]
```diff
@@ -0,0 +1,331 @@
...
+     */
+    amount: string
+
+    /**
+     * See [Tags](https://developers.unit.co/#tags).
+     */
+    tags?: object
+}
+
+export interface ACHPayment {
```
Comment (ilyamerman) [inline types/payments.ts:45]:
```suggestion
export interface AchPayment {
```

**Review by ilyamerman — CHANGES_REQUESTED** [review]
Everything is well, just "ACH" => "Ach" :)

**Pull request merged**